Packs/BitSight/ReleaseNotes/1_0_2.md

```diff
@@ -0,0 +1,27 @@
+
+#### Incident Fields
+- **BitSight Remaining Decay** - new fields instead of **Remaining Decay** field.
```

Comment:
same for all others.
```suggestion
- **Breaking changes**: The **BitSight Remaining Decay** new field was created instead of the **Remaining Decay** field.
```

Reply:
```suggestion
- **Breaking changes**: The new **BitSight Remaining Decay** field was created instead of the **Remaining Decay** field.
```